db/schema.rb

```diff
@@ -90,11 +90,15 @@
     t.string   "processing_state",        default: "requested", null: false
     t.datetime "created_at",                                    null: false
     t.datetime "updated_at",                                    null: false
-    t.string   "reference_no"
-    t.boolean  "closed"
     t.boolean  "override_delivery_error", default: false
     t.string   "delivery_error_type"
+    t.string   "reference_no"
+    t.boolean  "closed"
```

Comment:
I give up.  Thus it was decreed by Caesar that all numbers of reference numbers booleans that spaketh 'closed' shall return to place of their birth there to be counted by the order of the arbitrary men of Judea....